conf/iptables.conf.example

```diff
@@ -78,6 +78,8 @@
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 80  --jump ACCEPT
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 443 --jump ACCEPT
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 647 --jump ACCEPT
+# HTTP (parking portal)
+-A input-internal-vlan-if --protocol tcp --match tcp --dport 5252 --jump ACCEPT
```

Comment:
I agree with mister @fdurand. We should try to use constants / config parameters for all ports that are non-standards.
It is not THAT big of a deal on the side of iptables generation :)